yoti/admin/views/yoti/tmpl/default.php

```diff
@@ -18,6 +18,9 @@
                 <?php echo JText::_('Note: On the Yoti Dashboard the callback URL should be set to:'); ?>
                 <strong><?php echo JUri::root(false).'index.php?option=com_yoti&task=login'; ?></strong>
             </p>
+            <p>
+                <?php echo JText::_('Warning: User IDs provided by Yoti are only valid within the scope of the application. Changing your Yoti application will result in different Yoti user IDs.'); ?>
```

Comment:
How about:
"User IDs are unique to each Yoti Application.  Using a different Yoti Application means you will receive a different User ID for all of your users."

Reply:
I will use the same text here from Drupal PRs.